Add raid time check to automod_thread helper

oytundd opened the pull request:
Introduces logic to verify if the raid time has passed in raidhelper threads before applying moderation actions. Also comments out a logger statement in member_events.py to reduce log verbosity.(Mentioned in #402)
Closes: #403

oytundd committed:
Add raid time check to automod_thread helper

Introduces logic to verify if the raid time has passed in raidhelper threads before applying moderation actions. Also comments out a logger statement in member_events.py to reduce log verbosity.

brtran4 commented on moddingway/workers/helper.py:
```diff
@@ -75,10 +75,32 @@ async def automod_thread(
+        and thread.owner_id == user_id
+        and starter_message is not None
+    ):
+        date_pattern = re.compile(r"<t:(\d+):F>")
```
should probably comment what this regex is looking for

brtran4 replied:
er sorry i shouldve been more clear, what's the format of the timestamp? unix?

brtran4 commented on moddingway/events/member_events.py:
```diff
@@ -156,7 +156,7 @@ async def on_thread_create(thread: Thread):
                 logger.error(f"No message content found in event thread {thread.id}")
                 return
-            logger.info(message.content)
+            # logger.info(message.content)
```
if this is related to https://github.com/naurffxiv/moddingway/issues/402 we should link it in the commit message and delete this line

brtran4 commented on moddingway/workers/helper.py:
```diff
@@ -75,10 +75,32 @@ async def automod_thread(
+        if time_since_raid < (duration * 86400):  # 86400 seconds = 1 day
+            # logger.info(
+            #     f"Raid not started yet or duration not passed for thread {thread.id}"
+            # )
```
let's delete this comment

oytundd committed:
pr fixes

Cleaned up unused logger.info comments in member_events.py and helper.py.

oytundd committed:
Clarify comment about raid date extraction

Updated the comment to specify that the extracted raid date from the raidhelpers message is in Unix time format.
